resources/css/bem/wiki-search.less

```diff
@@ -20,7 +20,12 @@
     font-size: @font-size--wiki-search;
     background-color: @osu-colour-b6;
 
-    border: @_border-size solid @osu-colour-orange-1;
+    border: @_border-size solid @osu-colour-b6;
+    
+    &:focus-within {
+      box-shadow: 0 0 10px @osu-colour-orange-1;
+      border: @_border-size solid @osu-colour-orange-1
```

Comment:
the style doesn't quite match with beatmap search box (wrong border size, rounding, and colour). Also the colour should use page hue css variable, not actual orange.

Reply:
I have adjusted the border size and the rounding of the border. Do we prefer the thinner border? I've also changed the color to match the other elements on the page instead of the color of wiki-notice. In order to keep the code consistent I'm still using the pre-defined colors from color.less. 


![osu-css-border-style2](https://github.com/ppy/osu-web/assets/77649505/17cf0733-da72-4564-8faf-ff50dbaf27e3)